_lib/wordpress_post_processor.py

```diff
@@ -49,7 +49,9 @@ def process_post(post, newsroom = False):
         post['links'] = links
     else:
         post['tags'] = [tag['title'] for tag in post['taxonomy_fj_tag']]
-        post['author'] = [author['title'] for author in post['taxonomy_fj_author']]
+        post['author'] = [author['title'] for author in \
+            post['taxonomy_fj_author'] if 'Press Release' not in \
```

Comment:
I don't think the trailing `\` is necessary within the list comprehension's brackets. But they don't do any harm.